lib/node_modules/@stdlib/array/complex64/lib/main.js

```diff
@@ -917,6 +917,56 @@ setReadOnly( Complex64Array.prototype, 'get', function get( idx ) {
 	return new Complex64( buf[ idx ], buf[ idx+1 ] );
 });
 
+/**
+* Tests whether all elements in a array pass a test implemented by a predicate function.
+*
+* @name every
+* @memberof Complex64Array.prototype
+* @type {Function}
+* @param {Function} [predicate] - predicate - test function
```

Comment:
```suggestion
* @param {Function} predicate - predicate - test function
```

`predicate` is a mandatory argument.